src/components/molecules/heroSection/why-choose/WhyChooseSection.tsx

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import DefaultText from "components/atoms/typography/default-text";
+import Image from "next/image";
+import Twitter from "../../../../../public/assets/images/twitter.svg";
+import Circular from "../../../../../public/assets/images/circulate.svg";
+import Permission from "../../../../../public/assets/images/permission.svg";
+
+const WhyChooseSection = () => {
+  return (
+    <div className="mx-auto w-full max-w-screen-2xl bg-white">
+      <div className="m-10 pb-10">
+        <div className="mb-10 lg:w-[30%] ">
+          <DefaultText className="text-left font-inter text-4xl font-extrabold leading-[2.8rem]">
+            Why Choose TechIsHiring ?
+          </DefaultText>
+        </div>
+        <div className="grid grid-cols-1 gap-4 md:grid-cols-2 lg:grid-cols-3 ">
+          <div className="flex flex-col space-y-3 rounded-xl border p-6">
+            <div className="relative h-8 w-8 ">
+              <Image src={Twitter} alt="Twitter Icon" />
```

Comment:
Could you change this from using an imported image to using the images in the React Icons pack? Here's the site for the Icons: https://react-icons.github.io/react-icons

Please use the same Twitter Icon that we are using in the footer. Also please delete the image that was added to the public assets folder please.